### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by sungam3r**
Hey, this is a wonderful library for health checks! I needed to understand the work of this library for one project and I noticed some problems that I tried to fix. This PR does not apply to any of the issues described. I understand that usually it is recommended to start a new focused issue and the corresponding PR. But corrections for the most part relate to various minor fixes (forgotten cancellation tokens or calling `Dispose()`), typos, and formatting. I separated the PR into several rather small commits, grouping the changes in the files by meaning. I hope that viewing it in this form does not cause any troubles.

### CODE REVIEW

**Comment by unaizorrilla** [comment]
Hi @sungam3r 

Wow, this is big PR!! Probably I need some days to review and merge.

Thanks for send this!!

File: src/HealthChecks.Aws.S3/S3HealthCheck.cs [hunk 1]
```diff
@@ -10,8 +10,14 @@ namespace HealthChecks.Aws.S3
     public class S3HealthCheck : IHealthCheck
     {
         private readonly S3BucketOptions _bucketOptions;
+
```
Comment (unaizorrilla) [inline src/HealthChecks.Aws.S3/S3HealthCheck.cs]: I'm sorry but we not use blank lines for this, I review this PR but probably remove this blank lines. In fact, when I review code or perform some change I try to remove this blank lines.

Comment (sungam3r) [reply src/HealthChecks.Aws.S3/S3HealthCheck.cs]: > we not use blank lines for this

For what exactly? I see that the methods are universally separated by empty lines, but in some places there was no separation. I tried to keep the overall consistent style (with separation) wherever possible. In any case of course you are free to change it back. I was not primarily interested in changing code style.

Comment (unaizorrilla) [reply src/HealthChecks.Aws.S3/S3HealthCheck.cs]: Well, I use this practice from other OSS projects I contribute on. In my case, VS do  blank lines automatically to show code references.authors etc , this is enought for me!

Comment (sungam3r) [reply src/HealthChecks.Aws.S3/S3HealthCheck.cs]: Do you mean CodeLens?

Comment (unaizorrilla) [reply src/HealthChecks.Aws.S3/S3HealthCheck.cs]: yeap!

Comment (sungam3r) [reply src/HealthChecks.Aws.S3/S3HealthCheck.cs]: Then I understand. Although there is only a small problem ) :
изображение

I understand what kind of separation we are talking about - the separation at the visual level of the editor. But here the question is that not every editor has such functions.

Comment (unaizorrilla) [reply src/HealthChecks.Aws.S3/S3HealthCheck.cs]: Yes, I know! . I'm not closed to change this "convention". Let me close first this PR and I try to do a long/short conversation with other collaborators.

Thanks again!!

Comment (sungam3r) [reply src/HealthChecks.Aws.S3/S3HealthCheck.cs]: No problems, thanks.

**Comment by unaizorrilla** [comment]
Merged on master with no package update changes!
I don't know why but gh don't show this as merged and the commits are on master, probably my fault but I don't know what I do wrong in this PR.

Your commits are now on master https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/commits/master and I add you as contributor on Readme.md

**Pull request closed without merging**